src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1537,9 +1537,9 @@ public function qualifyColumn($column)
      * @param  array|\Illuminate\Database\Query\Expression  $columns
      * @return array
      */
-    public function qualifyColumns($columns)
+    public function qualifyColumns(...$columns)
```

Comment:
Unfortunately, even though this recreates the behaviour from Laravel 7, this is a breaking change. You may want to send this in to master.